### DISCUSSION TITLE
Conflicting values for resource 'Files/Uno.UI.Toolkit/Themes/Generic.xbf'

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#13980] [Q&A]
Hi guys, sorry it may be a spam merely. 

Is it truly WinUI issue in terms of poor error message ? https://github.com/microsoft/microsoft-ui-xaml/issues/8959

**Comment by limeniye** [comment]
I've tried it and I have the following stack of errors:

<h3>For Prism.Uno.WinUI "9.0.271-pre"</h3>


```
1>C:\Program Files\dotnet\sdk\8.0.100-rc.2.23502.2\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Windows.targets(55,5): error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll. Please update to a newer .NET SDK in order to reference this assembly.
1>Done building project "UnoApp29.csproj" -- FAILED.
```

<h3>For Prism.Dry.Ioc.Uno.WinUI "9.0.271-pre"</h3>


![image](https://github.com/unoplatform/uno/assets/69314237/2610695f-c2e3-4366-ba12-003b34a8ca94)

<h3>Options</h3>

Visual Studio: 17.8.0 Preview 3.0
WinUI: 5.0.0-dev.3533
WindowsAppSDK: 1.4.230913002
Net: 8.0.100-rc.2.23502.2 (latest)

I would like to draw attention to this problem @dansiegel 

[Prism.Uno.WinUI.zip](https://github.com/unoplatform/uno/files/12927499/Prism.Uno.WinUI.zip)

**Reply by sigmarsson** [reply]
Nuget restored failed; please try NET7 for size, rather the very latest.

**Comment by limeniye** [comment]
@sigmarsson, why do you need the Prism.Dry package?

You can try to use the [Uno Dependency Injection](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/DependencyInjection/HowTo-DependencyInjectionSetup.html): 
![image](https://github.com/unoplatform/uno/assets/69314237/95a7d247-d9c5-4757-86a5-fa1405775647)

**Reply by sigmarsson** [reply]
because my project had started prior to this UNO wizard released. Prism came with DryIoc, which I started to use.
